protocols/mdns/src/behaviour.rs

```diff
@@ -157,6 +207,7 @@ where
     }
 
     /// Expires a node before the ttl.
+    #[deprecated(note = "Unused API. Will be removed in the next release.")]
```

Comment:
More out of curiosity, how do you know this is unused? Given that it is `pub` any user could use it, no?

Reply:
Yes. I'd wait for people to raise that they are using it to learn about their usecase. I can't think off a usecase for it.